scripts/rabbitmq-server-ha.ocf

```diff
@@ -1619,6 +1640,45 @@ get_monitor() {
     return $rc
 }
 
+order_node_restart() {
+    local node=${1:?}
+    local nowtime
+    nowtime=$(now)
+    ocf_log warn "${LH} Ordering node '$node' to restart"
+    ocf_update_private_attr 'rabbit-ordered-to-restart' "$nowtime" "$node"
+}
+
+grep_partitions_report() {
+    local node="${1:?}"
+    local report="${2:?}"
+    local rabbit_node
+    rabbit_node=$(rabbit_node_name "$node")
+    echo "$report" | grep "PARTITIONED $rabbit_node:" | sed -e 's/^[^:]\+: //'
+}
+
+# Report partitions (if any) from viewpoint of every running node in cluster.
+# If node sees partition, report will contain the line like:
+#     PARTITIONED node-name: list-of-nodes, which-node-name-considers, itself-partitioned-with
+partitions_report() {
+    xargs -0 ${OCF_RESKEY_ctl} eval <<EOF
+RpcTimeout = 10,
```

Comment:
Most of that time is spent not on an RPC call itself, but on starting up `rabbitmqctl`. After that price is paid, it performs all requests in parallel, and they should take insignificant amount of time. At this point we are just trying to make our best effort, so I think 10 seconds is more than enough.